### PR TITLE
Ems development

### PR DESCRIPTION
**Pull request by swatikashyap2**

### CODE REVIEW

File: app/assets/javascripts/admin/users.coffee
Comment (akshitasandal1104) [inline app/assets/javascripts/admin/users.coffee:1]: Delete this file if not needed @swatikashyap2

Comment (akshitasandal1104) [inline app/assets/javascripts/admin/users.coffee:1]: Delete this file if not needed @swatikashyap2

Comment (akshitasandal1104) [inline app/assets/javascripts/admin/users.coffee:1]: Delete this file if not needed @swatikashyap2

File: app/assets/javascripts/application.js
Comment (akshitasandal1104) [inline app/assets/javascripts/application.js:1]: Delete this file if not needed @swatikashyap2

File: app/assets/javascripts/admin/users.coffee
Comment (swatikashyap2) [reply app/assets/javascripts/admin/users.coffee:1]: we have changed our rails version after that files structure also changed and these files are also deleted.

Comment (swatikashyap2) [reply app/assets/javascripts/admin/users.coffee:1]: we have changed our rails version after that files structure also changed and these files are also deleted.

File: app/assets/javascripts/application.js
Comment (swatikashyap2) [reply app/assets/javascripts/application.js:1]: we have changed our rails version after that files structure also changed and in that case these files are also deleted.

**Pull request merged**